### DISCUSSION TITLE
建议在5.0版本中用smux取代原本的mux

### DISCUSSION THREAD

**Discussion by maskedeken** [v2fly/v2ray-core#551] [Ideas]
建议在5.0版本中用smux取代原本的mux

**Comment by Loyalsoldier** [comment]
PR is welcomed.

**Comment by RPRX** [comment]
不需要 breaking，我的计划是对于 VLESS 和 VMess，XUDP 占 Command 4，SMUX 占 Command 5

SMUX 上用的子协议可以参考 Trojan-Go 的 SimpleSocks（看到你的 Xray 仓库中已实现了它）
